package.json

```
@@ -40,7 +39,7 @@
     "watchsass": "node_modules/.bin/node-sass $npm_package_assets_styles_sass --output $npm_package_assets_styles_css --output-style compressed --watch",
     "postsass": "for f in $npm_package_assets_styles_css; do file=${f%.css}; node_modules/.bin/cleancss -o $file.css $f; done",
     "makepot": "wpi18n makepot --domain-path languages --pot-file woocommerce-gateway-amazon-payments-advanced.pot --type plugin --main-file woocommerce-gateway-amazon-payments-advanced.php --exclude node_modules,tests,docs",
```

Exclude docker when generating translations file.
```suggestion
    "makepot": "wpi18n makepot --domain-path languages --pot-file woocommerce-gateway-amazon-payments-advanced.pot --type plugin --main-file woocommerce-gateway-amazon-payments-advanced.php --exclude node_modules,tests,docs,docker",
```

---

Added